terraform/helm/aptos-node/values.yaml

```diff
@@ -47,24 +82,41 @@ validator:
   affinity: {}
   # -- Validator configuration. See NodeConfig https://github.com/aptos-labs/aptos-core/blob/main/config/src/config/mod.rs
   config:
-    sync_only: false
-    concurrency_level: 8
-    round_initial_timeout_ms:
-    quorum_store_poll_count: 1
-    enable_state_store_pruner: true
-    enable_ledger_pruner: true
-    ledger_prune_window: 10000000
-    state_store_prune_window: 1000000
-    ledger_pruning_batch_size: 10000
-    state_store_pruning_batch_size: 10000
+    base: {}
+    consensus:
+      quorum_store_poll_count: 1
+    execution:
+      concurrency_level: 8
+    full_node_networks: []
+    inspection_service: {}
+    logger: {}
+    mempool: {}
+    metrics: {}
+    peer_monitoring_service: {}
+    api: {}
+    state_sync: {}
+    firehose_stream: {}
+    storage:
+      storage_pruner_config:
+        ledger_pruner_config:
+          enable: false
+          prune_window: 10_000_000
```

Comment:
Should we update this value to the default value in https://www.notion.so/aptoslabs/Blockchain-default-config-review-c0ec77c6fc4341b4a270e013eb4a33f7? Or just let it pick up the default value in Rust code?

Reply:
Does the default pruner config in Rust make sense for a production deployment?

Reply:
Yes